charts/aws-efs-csi-driver/values.yaml

```diff
@@ -94,6 +100,11 @@ controller:
     runAsUser: 0
     runAsGroup: 0
     fsGroup: 0
+  # securityContext on the controller container
+  containerSecurityContext:
```

Comment:
Setting privileged=false will cause the "delete-access-point-root-dir" controller option to fail. 

I'd like us to do a follow up PR where we add a quick comment mentioning that here, or, we can modify this PR to add that.

Reply:
Added a comment in this PR under https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/1152/commits/0fe7e6ce7c0b21b117afc3b2f5d07a7fcaf6da40